result/yoda.js

```diff
@@ -0,0 +1,74 @@
+if ("red" === color) {
+    // ...
+}
+if (color === "red") {
+    // ...
+}
+/*eslint yoda: "error"*/
+
+if ("red" === color) {
+    // ...
+}
+
+if (true == flag) {
+    // ...
+}
+
+if (5 > count) {
+    // ...
+}
+
+if (-1 < str.indexOf(substr)) {
+    // ...
+}
+
+if (0 <= x && x < 1) {
+    // ...
+}
+/*eslint yoda: "error"*/
+
+if (5 & value) {
+    // ...
+}
+
+if (value === "red") {
+    // ...
+}
+/*eslint yoda: ["error", "never", { "exceptRange": true }]*/
+
+function isReddish(color) {
+    return (color.hue < 60 || 300 < color.hue);
+}
+
+if (x < -1 || 1 < x) {
```

Comment:
__[development][eslint]__
* Error - Expected literal to be on the left side of <. (yoda)
* Error - 'x' is not defined. (no-undef)